core/install_gst.sh

```diff
@@ -3,6 +3,9 @@
 # exit when any command fails
 set -e
 
+# Check if the script is running as root
+[[ $EUID != 0 ]] && echo "Script must run as root."  && exit 1
```

Comment:
This is not part of this PR, a new PR should be created for it

Reply:
How did this even got here????